src/node/rpc/serialization.h

```diff
@@ -11,14 +11,14 @@ namespace ccf
 {
   DECLARE_JSON_ENUM(
     ccf::State,
-    {{ccf::State::uninitialized, "uninitialized"},
-     {ccf::State::initialized, "initialized"},
-     {ccf::State::pending, "pending"},
-     {ccf::State::partOfPublicNetwork, "partOfPublicNetwork"},
-     {ccf::State::partOfNetwork, "partOfNetwork"},
-     {ccf::State::readingPublicLedger, "readingPublicLedger"},
-     {ccf::State::readingPrivateLedger, "readingPrivateLedger"},
-     {ccf::State::verifyingSnapshot, "verifyingSnapshot"}})
+    {{ccf::State::uninitialized, "Uninitialized"},
```

Comment:
should this be ccf::State::UNINITIALIZED, "Uninitialized"?

Reply:
I'm not changing any C++ enum names in this PR, just serialization. But yes, this should eventually be changed.